extensions/extensions-data-delivery/aissemble-extensions-data-delivery-spark-py/src/data_delivery_spark_py/test_utils/spark_session_manager.py

```diff
@@ -47,7 +47,11 @@ def create_standalone_spark_session(sparkapplication_path: str) -> SparkSession:
             ",".join(spec_map["spec"]["deps"]["repositories"]),
         )
     for key, value in spec_map["spec"]["sparkConf"].items():
-        builder = builder.config(key, value)
+        logger.info("sparkConf - key: {}, value: {}: ".format(key, value))
+        if "spark.jars.ivy" != key.lower():
+            builder = builder.config(key, value)
+        else:
+            logger.info("Skip spark config - key: {}, value: {}: ".format(key, value))
```

Comment:
A: Can probably just print the key to be a little less noisy since we just printed the key and value above.